src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -421,6 +433,24 @@ private void processEnd(
         doEnd();
     }
 
+    private void processAbort(
+            DirectBuffer buffer,
+            int index,
+            int length)
+    {
+        factory.correlations.remove(acceptCorrelationId);
+        if (targetBeginIssued)
+        {
+            factory.writer.doAbort(target, targetId);
+            cleanupConnectReply.run();
```

Comment:
More strictly speaking, this line should be guarded by a condition that tests if the return value from `remove` above is `null`, i.e. already consumed by the connect reply.